test/e2e/admission_controller/admission_controller_suite_test.go

```diff
@@ -199,7 +199,7 @@ var _ = BeforeSuite(func() {
 
 	annotation = "ann-" + common.RandSeq(10)
 	yunikorn.EnsureYuniKornConfigsPresent()
-	yunikorn.UpdateConfigMapWrapper(oldConfigMap, "", annotation)
+	yunikorn.UpdateConfigMapWrapper(oldConfigMap, "")
```

Comment:
now annotation in Line 58 become unused, could you help remove it too ? Same thing in other places.

Reply:
done in b6c835adb62b7478e829cbe2e14f617209ab051e